content/docs/user-guide/assets/scene-settings/source-asset-best-practices.md

```diff
@@ -0,0 +1,84 @@
+
+If you have issues processing a scene source asset, check the [Event Log](../asset-processor/interface) in [Asset Processor](../asset-processor) for warnings and errors. The Event Log might contain information that can help you debug issues with scene source assets.
+
+## Meshes
+
+There are many options to configure mesh processing settings. Check the Scene Settings [Mesh Tab](../scene-settings/meshes-tab.md) topic for additional information.
+
+* **Mesh Density** - Although O3DE theoretically supports billions of vertices per mesh, ensure that your meshes have an appropriate density for your target platform.
+* **Clean mesh data** - Ensure the mesh data is coherent. Some DCC applications can create and export meshes with issues such as the following:
+  
+  * poly lines (a polygon with just two points)
+  * non-manifold geometry (edges that share more than two faces)
+  * globally small faces and edges
+  * unintended non-convex polygons
+  * self-intersecting polygons
+  * unintended open edges
+
+  Most DCC applications offer tools and scripts that you can use to check for these issues before exporting to an FBX scene source asset.
+
+* **Triangles and Quads** - Meshes be composed of three or four sided polygons when possible. N-gons are supported, however, n-gons might be triangulated either during export from a DCC application or processing in O3DE, and yield unintended results such as non-convex surfaces.
+* **Normals** - Ensure meshes in your source scene assets have custom normals if required. Normals can be generated for meshes when they are processed, however, the generated normals will be averaged and might not accurately represent some models such as hard surfaces with creased edges. Some DCC applications make a distinction between mesh *points* that describe mesh surface positions, and *vertices* that describe mesh surface texture atlases. For DCC applications that support both point and vertex attributes, normals should be applied as a point attribute.
+* **UVs** - Shaders included with O3DE support up to two UV sets. Support for additional UV sets can be added in custom shaders. Use simple names for UV sets such as `UV0`, `UV1`, and so on. For DCC applications that support both point and vertex attributes, UVs should be applied as a vertex attribute.
+* **Vertex Color Streams** - Vertex color streams can represent a color to be used in a shader, and can also be used to pack per vertex attributes such as constraints and mass values for cloth meshes. Use floating point color values and sensible names for vertex color streams to ensure you can easily determine which streams are intended to represent surface color and which streams represent other per vertex attributes.
```

Comment:
Do we need to cover default engine color space / color space considerations as well?

Reply:
Not for this topic. Theses are just vertex color attributes.